osu.Game/Screens/Play/Player.cs

```diff
@@ -251,7 +252,11 @@ private void load(OsuConfigManager config, OsuGameBase game, CancellationToken c
 
             dependencies.CacheAs(HealthProcessor);
 
-            InternalChild = GameplayClockContainer = CreateGameplayClockContainer(Beatmap.Value, DrawableRuleset.GameplayStartTime);
+            InternalChildren = new Drawable[]
+            {
+                new GlobalScrollAdjustsVolume(),
```

Comment:
Failing test shows that this being active in player context should depend on `OsuSetting.MouseDisableWheel` being false.

Reply:
Have fixed this and added extra test coverage of the paused state as well (5d1701469848f89410d84a220e065754f003a42b).